### DISCUSSION TITLE
Unable to upload picture for GitHub student developer pack verification

### DISCUSSION THREAD

**Discussion by P10swati** [community/community#57906] [GitHub Education]
There is only take image option and that too front facing as a result the image is blurry and the application request is declined . Please help !

**Comment by Abhishek-PratapSingh** [comment]
Same issue

**Reply by baysonfox** [reply]
Also having the same issue.

**Reply by JulioRios00** [reply]
Same here

**Comment by samus-aran** [comment]
Hi @P10swati  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know how you get on 👍

**Reply by VarenytsiaMykhailo** [reply]
https://support.github.com/ticket/personal/0/2513761

please, I have the same problem

**Reply by boomalator** [reply]
This is basically not true. We have been trying for months to get this to process:
1. Didn't like the official student card because the date isn't clear enough.
2. Didn't like the copy of the official learning plan (no explanation)
3. Got the principal to write a specific letter... 
    a) the page will not use the rear-facing camera on any iPhone we tried.
    b) the page will not use the webcam on the computer that works fine in gaming and zoom
    c) the page stays on "processing information" for HOURS after a file upload.
    
 Seems pretty broken.

**Reply by Zakaria-Elouali** [reply]
same F problem

**Reply by ghost** [reply]
please fix this github team

**Comment by boomalator** [comment]
You're on to something here.

 it seems github doesn't want us to actually get this access.

**Comment by bione-dev** [comment]
good
